share/config_files/common/vconsole.conf

```diff
@@ -0,0 +1,2 @@
+KEYMAP=us
+FONT=eurlatgr
```

Comment:
Also with https://github.com/rhinstaller/anaconda/commit/515f0679276369713eaed7d33f86904f9fb062db there should always be a correct font set for the currently selected language. So isn't this change redundant ?